theme/templates/pages/group.html

```diff
@@ -381,36 +391,37 @@ <h4><strong>{{ pendingrequest.invitation_to|contact }}</strong>
                     </div>
                 </div>
 
-                {% if profile_user.is_group_owner or profile_user.is_group_editor or profile_user.is_group_viewer %}
-                    <div role="tabpanel" class="tab-pane fade in active" id="resources">
+                <div role="tabpanel" class="tab-pane fade in active" id="resources">
                      <div class="row">
-                        <div class="col-lg-2" id="facets">
-                            <div class="panel panel-default">
-                                <div class="panel-heading">
-                                    <h4 class="panel-title">
-                                        <span class="glyphicon glyphicon glyphicon-user"></span>
-                                        <a data-toggle="collapse" class="accordion-toggle"
-                                           data-parent="#accordion" href="#filter-shared-by">&nbsp;Shared by</a>
-                                    </h4>
-                                </div>
-                                <div id="filter-shared-by" class="facet-list panel-collapse collapse in">
-                                    <div class="panel-body">
-                                        <div class="list-group">
-                                            <ul class="list-group inputs-group">
-                                                <i class="list-group-item no-items-found">No resources have been shared with this group yet.</i>
-                                            </ul>
-                                        </div>
-
-                                        <div id="grantors-list">
-                                            {% for res in group_resources %}
-                                                <span data-grantor-id="{{ res.grantor.id }}"
-                                                      data-grantor-name="{{ res.grantor|best_name }}"></span>
-                                            {% endfor %}
-                                        </div>
-                                    </div>
-                                </div>
-                            </div>
-                        </div>
+                         {% if profile_user.is_group_owner or profile_user.is_group_editor or profile_user.is_group_viewer %}
```

Comment:
Don't show resources if not a Public Group and the user viewing the page does not have access